internal/backend/local/test.go

```diff
@@ -1191,6 +1198,25 @@ func (runner *TestFileRunner) getVariablesFromConfiguration(knownVariables terra
 	return variableValues, diags
 }
 
+// GetGlobalAndSuiteVariables returns the global and suite level variables
+func (runner *TestFileRunner) GetGlobalAndSuiteVariables(config *configs.Config) (terraform.InputValues, tfdiags.Diagnostics) {
+	var diags tfdiags.Diagnostics
+	values := make(terraform.InputValues)
+	// TODO: Verify it's not problematic that we don't care about the variables being used
```

Comment:
So we do care about them being used, but only because the types are ambiguous until they are referenced.

The `variable` blocks in Terraform provide the `type` functionality, and we do conversion into those types if/when they are referenced. We can't always parse the raw HCL if we don't know what type to parse it into.